medusa/post_processor.py

```diff
@@ -287,6 +288,16 @@ def list_associated_files(self, file_path, base_name_only=False, subtitles_only=
 
         return file_path_list
 
+    @staticmethod
+    def rar_basename(filepath, filelist):
+        videofile = os.path.basename(filepath)
+        rars = (x for x in filelist if rarfile.is_rarfile(x))
+
+        for rar in rars:
+            content = rarfile.RarFile(rar).namelist()
```

Comment:
Already told you. Will explain one more time.
The function is called with videoInRar. Which is the video extracted from RAR.